pkg/providers/tinkerbell/validate.go

```diff
@@ -64,6 +64,21 @@ func validateUpgradeRolloutStrategy(spec *ClusterSpec) error {
 	return nil
 }
 
+func validateAutoScalerDisabledForInPlace(spec *ClusterSpec) error {
+	cpUpgradeRolloutStrategyType := spec.ControlPlaneConfiguration().UpgradeRolloutStrategy
+	// We do not support different strategy types for Inplace between CP and worker nodes so it is okay to check only CP
```

Comment:
Do we already have a validation somewhere that ensures upgrade strategy is the same for CP and all wng?

Reply:
Yes, `validateUpgradeRolloutStrategy` takes care of that validation!